package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@graphql-validate/cli",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

no need to bump this manually 
```suggestion
  "version": "0.2.0",
```